css/popup.css

```diff
@@ -30,23 +30,6 @@ body {
     padding: 0 15px;
 }
 
-.btn-file {
-    position: relative;
-    overflow: hidden;
-}
-
-.btn-file input[type=file] {
-    position: absolute;
-    top: 0;
-    right: 0;
-    min-width: 100%;
-    min-height: 100%;
-    font-size: 100px;
-    text-align: right;
-    filter: alpha(opacity=0);
-    opacity: 0;
-    outline: none;
-    background: white;
-    cursor: inherit;
-    display: block;
+#files {
+    display: none;
```

Comment:
@J2TeaM Because I tested when I reply to you,so after work day I will check it once again.And I sent to you a message, Have a mistake or missing message here?